dashboard/app/views/levels/editors/fields/_droplet.html.haml

```diff
@@ -27,6 +27,12 @@
     = f.label :start_libraries,'User-Created Toolbox Libraries (make sure that Enable Libraries ^ is checked)'
     = f.text_area :start_libraries, value: @level.start_libraries
 
+  %h1.control-legend{data: {toggle: "collapse", target: "#ai"}}
```

Comment:
It looks like this field will show up on all droplet level types. It sounds like we only want it on App Lab?

Are there other editor settings you plan to add to this section? Otherwise I would move it up to where makerlab_enbaled is above in this file and reducing it down to the one line checkbox

Reply:
Oh, yeah, I can move the field to App Lab only. Good catch.

There are other settings we plan to add to this section. For example, in the future, a level builder will be able to include a pre-trained model. Most of these will come post-pilot, though.